src/status_im2/contexts/activity_center/view.cljs

```diff
@@ -214,15 +214,16 @@
     (rf/dispatch [:activity-center.notifications/fetch-first-page])
     (fn []
       (let [notifications (rf/sub [:activity-center/notifications])]
-        [rn/view {:flex 1 :padding-top (navigation/status-bar-height)}
-         [blur/view style/blur]
-         [header]
-         [rn/flat-list
-          {:data                      notifications
-           :render-data               active-swipeable
-           :content-container-style   {:flex-grow 1}
-           :empty-component           [empty-tab]
-           :key-fn                    :id
-           :on-scroll-to-index-failed identity
-           :on-end-reached            #(rf/dispatch [:activity-center.notifications/fetch-next-page])
-           :render-fn                 notification-component}]]))))
+        [theme/provider {:theme :dark}
```

Comment:
Is it okay that we're hardcoding the dark theme here?

Reply:
I think in general yes, it's not so different from setting `override-theme :dark` to each component really. Perhaps an issue if those aren't aware of the theme provider approach, I will add some docs on this/ figure out how to communicate it out 👍